doc/services/device_mgmt/ota.rst

```diff
@@ -36,27 +47,26 @@ upgrade occurs.
 2. The `Golioth OTA documentation`_ includes complete information about the
    versioning process
 
-Eclipse hawkBit |trade|
+Lightweight M2M (LWM2M)
 =======================
 
-`Eclipse hawkBit`_ |trade| is an update server framework that uses polling on a
-REST api to detect firmware updates. When a new update is detected, the binary
-is downloaded and installed. MCUboot can be used to verify the signature before
-upgrading the firmware.
+The :ref:`lwm2m_interface` protocol includes support for firmware update via
+:kconfig:option:`CONFIG_LWM2M_FIRMWARE_UPDATE_OBJ_SUPPORT`. Devices securely
+connect to an LwM2M server using DTLS. A :zephyr:code-sample:`lwm2m-client` sample is
+available but it does not demonstrate the firmware update feature.
 
-There is a :zephyr:code-sample:`hawkbit-api` sample included in the
-Zephyr :ref:`mgmt-samples` section.
+Memfault
+========
 
-UpdateHub
-=========
+`Memfault`_ is a IoT observability platform that includes OTA management. Devices
+check-in with Memfault's service periodically for an OTA update, and when an update is
+available, download and install the binary.
 
-`UpdateHub`_ is a platform for remotely updating embedded devices. Updates can
-be manually triggered or monitored via polling. When a new update is detected,
-the binary is downloaded and installed. MCUboot can be used to verify the
-signature before upgrading the firmware.
-
-There is an :zephyr:code-sample:`updatehub-fota` sample included in the Zephyr
-:ref:`mgmt-samples` section.
+Zephyr projects that use MCUboot and have a direct internet connection can leverage Memfault's
+built-in `Memfault OTA client`_ to download a payload from Memfault's OTA service, load it into
+the secondary partition, and then reboot into the new image. A working sample that
+leverages this client is available in the `Memfault Zephyr ESP32 sample`_ repository.
+`Memfault's OTA documentation`_ provides more information on how to deploy and manage releases.
```

Comment:
is there any reason this can't be moved to the zephyr repo?

Reply:
That is a great idea! What is the process for adding a sample app to Zephyr?

Reply:
I would love to get this app into zephyr but I may need some guidance as it relies on a currently external module (Memfault sdk). If we add this sample to the zephyr app, perhaps we would also want to add Memfault to zephyr's `west.yml`? Unsure what the process is there so I'm very open to your suggestions & guidance!

Reply:
Would have to read and follow this: https://docs.zephyrproject.org/latest/contribute/external.html would need to submit an issue with the details of the module for discussion with TSC

Reply:
Great, thank you! I can certainly do that. Do you consider that a prerequisite of this PR or is it fine to add the documents with the out-of-tree sample app, and update them when it is added?

Reply:
I would rather not personally, we have samples in tree and this should be there and available for all platforms. Having an external repo whereby it works for one board/SoC? That's not really useful in my mind